workflows4s-quartz/src/test/scala/workflows4s/runtime/wakeup/quartz/QuartzKnockerUpperTest.scala

```diff
@@ -23,11 +24,27 @@ class QuartzKnockerUpperTest extends AnyFreeSpec with Matchers with BeforeAndAft
     "should schedule a wakeup at a specified time" in withQuartzKnockerUpper { knockerUpper =>
       var wokenUpAt: Instant = null
       val testId             = "test-id"
-      knockerUpper.initialize(id => IO { if (id == testId) wokenUpAt = Instant.now() }).unsafeRunSync()
-      val wakeupAt           = Instant.now().plusMillis(100)
+      knockerUpper
+        .initialize(id =>
+          IO {
+            if (id == testId) {
+              wokenUpAt = Instant.now()
+              println(s"Woken up at $wokenUpAt") // Add debug logging
+            }
+          },
+        )
+        .unsafeRunSync()
+      val wakeupAt           = Instant.now().plusMillis(500) // Increased delay to 500ms
+      println(s"Scheduling wakeup at $wakeupAt") // Add debug logging
       knockerUpper.updateWakeup(testId, Some(wakeupAt)).unsafeRunSync()
+      // Patience config is already imported at the top of the file
+      implicit val patience: PatienceConfig = PatienceConfig(
+        timeout = Span(10, Seconds), // Increased timeout
```

Comment:
I think this is too much. I wouldn't expect this to take more than few seconds on even the slowest but modern machines. If we bump it too much and test starts failing the execution time will grwo quite a lot. Its not that important here (just a single test being impacted) but its a good practice in general - it might have huge impact if there are tens or hundreds of tests.

I suggest 1-2s as a starting point. Maybe 5 if needed.